src/Raven.Studio/wwwroot/App/views/database/indexes/editIndex.html

```diff
@@ -102,6 +102,17 @@ <h2 id="reduce-title" class="margin-bottom margin-bottom-xs on-base-background">
                         <input type="text" class="form-control flex-grow has-error" placeholder="Enter collection name" data-bind="textInput: editedIndex().reduceToCollectionName, visible: editedIndex().outputReduceToCollection">
                     </div>
                 </div>
+                <div class="flex-horizontal margin-bottom" data-bind="visible: editedIndex().outputReduceToCollection">
```

Comment:
is it only visible when output reduce to collection is set to true?

Reply:
yes, that's correct